app/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,8 @@
     <string name="error_wallet_not_found_title">No wallet found</string>
     <string name="error_wallet_not_found_message">Try account or use an another phone number</string>
     <string name="error_too_often_otp_requests_message">You request SMS too often. Try again in %1d sec.</string>
+    <string name="error_shares_does_not_matches_title">Shares aren\'t\nmatch</string>
```

Comment:
Incorrect translation. 
Should be `Shares aren't matching` or `Shares do not match`.
Please talk to our product manager about this translation